### PR TITLE
Fixes #13: Upgrade react-jsonschema-form to 0.12

### LINKED ISSUE #13: Upgrade react-jsonschema-form to 0.12.0
Changelog: https://github.com/mozilla-services/react-jsonschema-form/releases/tag/v0.12.0

As we're starting from 0.4 currently, expect some breakage. /cc @magopian

### PR DESCRIPTION
**Pull request by n1k0**
r=? @magopian (let's pair review this tomorrow)

### CODE REVIEW

File: css/rjsf.css [hunk 1]
```diff
@@ -1,25 +1,25 @@
-.generic-form * {
+.rjsf * {
```
Comment (magopian) [inline css/rjsf.css]: is `rjsf` a fake/random name just for the tests, as I believe it is? If so, I'd rather have something like `random-name` or `test-name` or even `foobar` so it's explicit ;)

**Comment by magopian** [comment]
a very small nit, other than that, r+

Thanks!

File: css/rjsf.css (hunk 1 above)
Comment (n1k0) [inline css/rjsf.css]: Nope, it's acronym for *r*eact *j*son *s*chema *f*orm and is the name of the provided CSS stylesheet root identifier for the built forms.

Comment (magopian) [inline css/rjsf.css]: ah, makes total sense now that I see it, thanks for the explanation!

**Pull request merged**